tests/gui/test_widgets.py

```diff
@@ -3052,21 +3052,20 @@ def test_FileWidget_init_file_not_downloaded(mocker, source, session):
     session.commit()
 
     get_file = mocker.MagicMock(return_value=file)
-    with threads(1) as [export_thread]:
-        controller = mocker.MagicMock(get_file=get_file, export_thread=export_thread)
+    controller = mocker.MagicMock(get_file=get_file)
 
-        fw = FileWidget(
-            "mock", controller, mocker.MagicMock(), mocker.MagicMock(), mocker.MagicMock(), 0, 123
-        )
+    fw = FileWidget(
+        "mock", controller, mocker.MagicMock(), mocker.MagicMock(), mocker.MagicMock(), 0, 123
+    )
 
-        assert fw.controller == controller
-        assert fw.file.is_downloaded is False
-        assert not fw.download_button.isHidden()
-        assert not fw.no_file_name.isHidden()
-        assert fw.export_button.isHidden()
-        assert fw.middot.isHidden()
-        assert fw.print_button.isHidden()
-        assert fw.file_name.isHidden()
+    assert fw.controller == controller
+    assert fw.file.is_downloaded is False
+    assert not fw.download_button.isHidden()
+    assert not fw.no_file_name.isHidden()
+    assert fw.export_button.isHidden()
+    assert fw.middot.isHidden()
+    assert fw.print_button.isHidden()
+    assert fw.file_name.isHidden()
```

Comment:
This diff looks huge, but the only thing that happened is that the `export_thread` is no longer passed as an argument when constructing the `Controller` instance. Because threads require management, this causes some indentation changes :slightly_frowning_face: